js/notify/electron-notify-preload.js

```diff
@@ -134,6 +134,26 @@ function setContents(event, notificationObj) {
 
     const winId = notificationObj.windowId;
 
+    if (!notificationObj.sticky) {
+        container.addEventListener('mouseleave', onMouseLeave);
```

Comment:
do you need to unhook these events somewhere - does this cause memory leak by holding ref to container???

Reply:
@lneir Done 😃 Please review 😃 Thanks